src/styles/_components/_modals.scss

```diff
@@ -37,7 +37,8 @@
         right: 24px;
     }
 
-    &__close-button--light svg {
+    &__close-button--light svg,
+    &__close-button--light .st0 {
```

Comment:
I could after I change that particular svg in all platforms, until then I have to include this. Which reminds me that we really should have an automated process to clean-up SVG's as this is starting to get annoying.